mac/KeymanEngine4Mac/KeymanEngine4Mac/KME/KMBinaryFileFormat.h

```diff
@@ -182,6 +182,11 @@ struct COMP_KEY {
 
 /* KVK file format definitions */
 
+#define KVKH_102                0x01
+#define KVKH_DISPLAY_UNDERLYING 0x02
```

Comment:
I see that these aren't the only definitions using these values; `#define KVKH_BITMAP` is also assigned `0x01` just a few lines later.

I assume that they belong to mutually-exclusive sets, but those sets aren't exactly explicit within the file.  Maybe a comment on the range / permissible values of each set?

Reply:
It's `KVKK_BITMAP` 😉  but will add comments